tests/crashes/119272.rs

```diff
@@ -1,27 +0,0 @@
-//@ known-bug: #119272
-#![feature(type_alias_impl_trait)]
-mod defining_scope {
-    use super::*;
-    pub type Alias<T> = impl Sized;
-
-    pub fn cast<T>(x: Container<Alias<T>, T>) -> Container<T, T> {
-        x
-    }
-}
-
-struct Container<T: Trait<U>, U> {
-    x: <T as Trait<U>>::Assoc,
-}
-
-trait Trait<T> {
-    type Assoc;
-}
-
-impl<T> Trait<T> for T {
-    type Assoc = Box<u32>;
-}
-impl<T> Trait<T> for defining_scope::Alias<T> {
-    type Assoc = usize;
-}
-
-fn main() {}
```

Comment:
Could you please put something like  Fixes #abcde   into the commit message so that the corresponding tickets are closed automatically? Thx!

Reply:
IIRC putting them into the PR is generally better, because that way GH spams less with cross references

Reply:
:+1: or that, sure :)

Reply:
I put them into the PR description 😁